packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -206,7 +212,14 @@ class Metrics {
         canProceed: true,
         identifiers,
         intervals: [options.intervalData],
-        eventData: {webClientDomain: window.location.hostname}
+        eventData: {webClientDomain: window.location.hostname},
+        sourceMetadata: {
+          applicationSoftwareType: 'webex-js-sdk',
+          applicationSoftwareVersion: 'x.x.x.x', // SDK version
```

Comment:
```suggestion
          applicationSoftwareVersion: this.webexVersion
```
version is available on the metrics object